file.c

```diff
@@ -6537,6 +6537,142 @@ const char ruby_null_device[] =
  *  read-only, which is reported as <code>0444</code>.
  *
  *  Various constants for the methods in File can be found in File::Constants.
+ *
+ *  == What's Here
+ *
+ *  First, what's elsewhere. \Class \File:
+ *
+ *  - Inherits from class IO -- in particular, methods for creating,
+ *    reading, and writing files.
+ *  - Includes module FileTest.
+ *
+ *  \Class \File provides methods for:
+ *
+ *  === Creating
+ *
+ *  - ::new:: Creates a new file at the given file path; returns the file.
+ *  - ::open:: Creates a new file at the given file path;
+ *             calls the block with the file.
+ *  - ::mkfifo:: Returns the FIFO file created at the given path.
+ *
+ *  === Deleting
+ *
+ *  - ::delete:: Deletes the file for each given file path.
+ *  - ::truncate:: Truncates the file at the given file path to the given size.
+ *  - ::unlink:: Deletes the file for each given file path.
+ *  - #truncate:: Truncates +self+ to the given size.
+ *
+ *  === Querying
+ *
+ *  _Paths_
+ *
+ *  - ::absolute_path:: Returns the absolute file path for the given path.
+ *  - ::absolute_path?:: Returns whether the given path is the absolute file path.
+ *  - ::basename:: Returns the last component of the given file path.
+ *  - ::dirname:: Returns all but the last component of the given file path.
+ *  - ::expand_path:: Returns the absolute file path for the given path.
+ *  - ::extname:: Returns the file extension for the given file path.
+ *  - ::fnmatch? (aliased as ::fnmatch):: Returns whether the given file path
+ *                                        matches the given pattern.
+ *  - ::path:: Returns the string representation of the given path.
+ *  - ::realdirpath:: Returns the real directory path for the given file path.
+ *  - ::realpath:: Returns the real path for the given file path.
+ *  - #path::  Returns the string representation of the given path.
+ *  - #to_path:: Returns the file path that was used to create the file.
+ *
+ *  _Times_
+ *
+ *  - ::atime:: Returns a Time object giving the time of the most recent access
```

Comment:
`Returns a \Time for the most recent access to the given file`. `\Time instance` or `\Time object` is also fine, but in general in Ruby you would say a method returns a time instead of a time object or a time instance.  Please use a consistent format for other entries in this section.